packages/foundry-sdk-generator/src/generate/betaClient/generatePackageJson.ts

```diff
@@ -22,32 +22,52 @@ export async function generatePackageJson(options: {
   packageVersion: string;
   packagePath: string;
   dependencies?: Array<{ dependencyName: string; dependencyVersion: string }>;
+  beta: boolean;
 }) {
+  const packageDeps = options.dependencies?.reduce((acc, value) => {
+    acc[value.dependencyName] = value.dependencyVersion;
+    return acc;
+  }, {} as { [dependencyName: string]: string });
+
   // Note that any "default" conditions _must_ be last in their block otherwise it will crash at runtime
-  const packageJson = {
-    name: options.packageName,
-    version: options.packageVersion,
-    main: "./index.js",
-    types: "./index.d.ts",
-    exports: {
-      ".": {
-        types: "./index.d.ts",
-        script: {
-          types: "./dist/bundle/index.d.ts",
-          default: "./dist/bundle/index.esm.js",
+  const packageJson = options.beta
+    ? {
+      name: options.packageName,
+      version: options.packageVersion,
+      main: "./index.js",
+      types: "./index.d.ts",
+      exports: {
+        ".": {
+          types: "./index.d.ts",
+          script: {
+            default: "./dist/bundle/index.esm.js",
+          },
+          default: "./index.js",
         },
-        default: "./index.js",
       },
-      "./ontology/objects": {
-        types: "./ontology/objects/index.d.ts",
-        default: "./ontology/objects/index.js",
+      dependencies: packageDeps,
+    }
+    : {
```

Comment:
The fallback block is unchanged right?

Reply:
Yes, besides me pulling out the dependencies into that helper function